docs/product/base/position.html

```diff
@@ -84,7 +84,9 @@
 <div class="ps-absolute t48 r24"> … </div>
 <div class="ps-absolute t64 l64"> … </div>
 <div class="ps-absolute bn8 rn8"> … </div>
+<div class="ps-absolute i48"> … </div>
 {% endhighlight %}
+<!-- TODO: Add inset example -->
```

Comment:
If we get the z-index right, this should be just fine to slap back there.

Reply:
Sticking it in the back of class is probably the way to go. I'll add this example in so we can get this sucker merged.